.cirrus.yml

```
@@ -132,13 +136,27 @@ macOS_task:
     - pyenv rehash
     - pip install --upgrade-strategy eager -U pip wheel setuptools
     - pip install --upgrade-strategy eager -U -r requirements-tests.txt
-    - pip install -e .
+    - pip install .
 
   script:
     - python3 --version
     - pip list
     - ./test.sh
 
+macOS_version_task:
+  skip: $CIRRUS_BRANCH =~ '.*\.tmp'
```

We probably don't want to run that on all branches etc.

```suggestion
  skip: $CIRRUS_BRANCH =~ 'canon'
```